R/nested.R

```diff
@@ -29,10 +29,10 @@ compute_parse_data_nested <- function(text) {
     add_terminal_token_before() %>%
     add_terminal_token_after()
 
+  parse_data$child <- rep(list(NULL), length(parse_data$text))
+  parse_data$short <- substr(parse_data$text, 1, 5)
```

Comment:
We don't use it further no. I It just helps when working interactively. Should we drop it?